joomla/joomla-cms
# [5.1] Schemaorg => Generic plugin

Pull Request for Issue # .

### Summary of Changes
This PR adds a "Generic" schema.org type which offers a JSON field to add any schema.org JSON-LD.


### Testing Instructions
- Apply the PR
- Go to an article (or create one)
- Setup the schema.org plugin once
- Select "Generic" as Schema Type
![grafik](https://github.com/joomla/joomla-cms/assets/1229869/4d8785e3-295b-44c4-8c8d-33b2e2aa2a33)
- Add any valid JSON-LD string to the JSON field
- Save
- Open article in frontend
- Check browser source code
- Check page with https://search.google.com/test/rich-results


### Actual result BEFORE applying this Pull Request
No Generic type available


### Expected result AFTER applying this Pull Request
Generic type available
